content/5_Plat/RURQ.mdx

```diff
@@ -37,15 +37,77 @@ queries.
 
 ### Implementation
 
-<Resources>
-	<Resource
-		source="Benq"
-		title="BITrange"
-		url="https://github.com/bqi343/USACO/blob/master/Implementations/content/data-structures/1D%20Range%20Queries%20(9.2)/BITrange.h"
-	/>
-</Resources>
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+class fenwick_tree {
```

Comment:
use the bit from the [purs module](https://usaco.guide/gold/PURS) for consistency

Reply:
Done